code-of-conduct.md

```diff
@@ -0,0 +1,128 @@
+# Contributor Covenant Code of Conduct
```

Comment:
Instead of this, grab or link to the OpenSSF CoC: https://openssf.org/community/code-of-conduct/

Reply:
Maybe add, at the end, a link to the OpenSSF COC?